bluepyopt/ephys/parameters.py

```diff
@@ -128,8 +128,8 @@ def instantiate(self, sim=None, icell=None):
         """Instantiate"""
         if self.value is None:
             raise Exception(
-                'NrnSectionParameter: impossible to instantiate parameter '
-                'without value')
+                'NrnSectionParameter: impossible to instantiate parameter "%s" '
```

Comment:
re-collapsing this logic back into a helper method is probably still a good idea